### ISSUE TITLE
Crash on Load

### ISSUE THREAD

**Issue by Dragnier** [abused/Technomancy-2#6]
**Versions**
---
* Technomancy 2: 1.1.0
* Blood Magic: 2.3.3-101
* Forge: 14.23.5.2806
* MC: 1.12.2

[Here is the crash log.](https://paste.dimdev.org/zikujovupa.mccrash)

**Comment by matyklug18** [comment]
i had the same issue, but i installed redstone flux and now it works.

**Comment by Dragnier** [comment]
That's still a bug if dependency on Redstone Flux wasn't intentional. It does occasionally happen accidentally. However, if it was intentional, it should be marked as a dependency on curse so it downloads when you download this mod.

**Comment by matyklug18** [comment]
Yes, it is a bug,i was just trying to help you temporaly fix the bug.
